packages/bull-vault/src/BullStrategy.sol

```diff
@@ -17,83 +24,111 @@ contract BullStrategy is ERC20, LeverageBull, UniBull {
     using StrategyMath for uint256;
 
     uint256 private constant ONE = 1e18;
+    uint32 private constant TWAP = 420;
```

Comment:
Are we sure we want to stick with this TWAP? CC @aleone

Reply:
i think twap mostly won't matter for this contract, only initial deposits, so not super critical

Reply:
This is removed